apps/docs/src/index.data.ts

```diff
@@ -65,7 +65,9 @@ export default defineLoader({
       const fileExist = watchedFiles.some((file) => file.endsWith(`${componentName}.stories.ts`));
       return {
         status: fileExist ? "in-progress" : "planned",
-        href: fileExist ? `/development/components/${componentName}` : undefined,
+        href: fileExist
+          ? `/development/components/${componentName.replace("Onyx", "")}`
```

Comment:
We need to full component name for the `fileExist` above :)
I would leave it like that because we will remove it anyways soon when we just link to Storybook